package-lock.json

```diff
@@ -50,7 +50,7 @@
 			}
 		},
 		"apps/backend": {
-			"version": "1.5.0",
```

Comment:
If you do not install any new dependencies package-lock,json file must contain no changes